src/traces/sankey/render.js

```diff
@@ -628,13 +735,31 @@ function snappingForce(sankeyNode, forceKey, nodes, d) {
             maxVelocity = Math.max(maxVelocity, Math.abs(n.vx), Math.abs(n.vy));
         }
         if(!d.interactionState.dragInProgress && maxVelocity < 0.1 && d.forceLayouts[forceKey].alpha() > 0) {
-            d.forceLayouts[forceKey].alpha(0);
+            d.forceLayouts[forceKey].alpha(0); // This will stop the animation loop
         }
     };
 }
 
 // basic data utilities
 
+function persistFinalNodePositions(d, gd) {
+    var x = [];
+    var y = [];
+    for(var i = 0; i < d.graph.nodes.length; i++) {
+        var nodeX = (d.graph.nodes[i].x0 + d.graph.nodes[i].x1) / 2;
+        var nodeY = (d.graph.nodes[i].y0 + d.graph.nodes[i].y1) / 2;
+        x.push(nodeX / d.figure.width);
+        y.push(nodeY / d.figure.height);
+    }
+    Plotly._guiRestyle(gd, {
```

Comment:
Strange that this doesn't result in circular dependency loop.

In other modules, we usually `var Registry = require('../../registry')` and use `Registry.call('_guiRestyle', /**/)`

Reply:
Mmm :thinking: should I change it?

Reply:
I would prefer, yes. Thanks!

Reply:
Sure thing! Done in 40fb223fe48cacb155f294193fedbffc15a93069